pkg/core/xds/types.go

```diff
@@ -57,10 +56,13 @@ type HealthCheckMap map[ServiceName]*mesh_core.HealthCheckResource
 // FaultInjectionMap holds the most specific FaultInjectionResource for each InboundInterface
 type FaultInjectionMap map[mesh_proto.InboundInterface]*mesh_proto.FaultInjection
 
+// TrafficPermissionMap holds the most specific TrafficPermissionResource for each InboundInterface
+type TrafficPermissionMap map[mesh_proto.InboundInterface]*mesh_core.TrafficPermissionResource
```

Comment:
Why not `mesh_proto.TrafficPermission`?

Reply:
We use the name of the resource. Check out `createRbacRule` function